client/src/components/questions_answers/QuestionsAnswers.jsx

```diff
@@ -167,17 +181,16 @@ class QuestionsAnswers extends React.Component {
       <div className='QA'>
         <div id='Qtitle'>QUESTIONS AND ANSWERS</div><br></br>
 
-        <form id='Qsearch'>
-          <input id='searchbar' type='text' placeholder='HAVE A QUESTION? SEARCH FOR ANSWERS...'></input>
-          <button type='submit' id='searchbtn' >Search</button>
-        </form><br></br>
+        <Search submitSearch={this.submitSearch.bind(this)} /><br></br>
+
+        <QuestionList questions={this.state.activeSearch ? this.state.filteredQuestions : this.state.questions} length={this.state.Qlength} displayMoreAnswers={this.displayMoreAnswers.bind(this)} likeAnswer={this.likeAnswer.bind(this)} likeQuestion={this.likeQuestion.bind(this)} reportQuestion={this.reportQuestion.bind(this)} reportAnswer={this.reportAnswer.bind(this)} aModalDisplay={this.aModalDisplay.bind(this)} />
```

Comment:
nice use of variable names! I don't know much about the Q&A widget but I can definitely tell what each variable is for.